packages/grafana-alerting/package.json

```diff
@@ -53,17 +69,22 @@
     "react": "18.3.1",
     "react-dom": "18.3.1",
     "react-redux": "^9.2.0",
+    "rimraf": "^6.0.1",
+    "rollup": "^4.41.1",
     "type-fest": "^4.40.0",
     "typescript": "5.7.3"
   },
   "peerDependencies": {
-    "@grafana/runtime": "^12.0.0-pre",
-    "@grafana/ui": "^12.0.0-pre",
+    "@grafana/runtime": "^11.6.2",
+    "@grafana/ui": "^11.6.2",
     "@reduxjs/toolkit": "^2.8.0",
+    "msw": "^2.0.0",
     "react": "^18.0.0",
     "react-dom": "^18.0.0"
   },
   "dependencies": {
+    "@faker-js/faker": "^9.8.0",
+    "fishery": "^2.3.1",
```

Comment:
I was under the impression these dependencies are used for tests. Should they be moved to dependencies so consumers don't needlessly download them?

Reply:
The initial idea here was to export fakes / mocks from this package so plugin authors can test their alerting integration(s) without having to be intimately familiar with the data structures of the API responses.

We currently use a similar MSW + fishery combo for integrations tests in Grafana core and would love to extend this to sibling projects like SLO / OnCall / Service Center, etc.

Any thoughts on how to do this in a better way? I tried `peerDependencies` but seemed to throw some weird errors when consuming the package in a demo plugin. I'll give this another try!